src/arrayMethodSort.js

```diff
@@ -3,9 +3,30 @@
 /**
  * Implement method Sort
  */
+/*
+realise sort method for array
+which repeats the original sort method
+the beginning is already there
+*/
+
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compFn = compareFunction || ((a, b) => String(a) > String(b));
```

Comment:
it's better to set as a default value for compare function params 
```
[].__proto__.sort2 = function(
  compareFunction = (a, b) => String(a) > String(b))
) {
```

Reply:
done